File: API/Entities/Poligon.cs
using System;

namespace API.Entities;

public class Tocka
{
    public decimal? Lat { get; set; }
    public decimal? Lng { get; set; }
}

public class TockaOboda
{
    public int? Zaporedje { get; set; }
    public Tocka? Tocka { get; set; }

}

public class Poligon 
{
    public int IdJavnegaObjekta { get; set; }
    public string? SifraJavnegaObjekta { get; set; }
    public string? Naziv { get; set; }


    public List<TockaOboda>? TockeOboda { get; set; }
    public List<List<Tocka>>? NoviObodiObjekta { get; set; }

}

File: client/src/app/_models/poligon.ts
export interface Itocka {
    lat: number;
    lng: number;
  }
  
  export interface ItockaOboda {
    zaporedje: number;
    tocka: Itocka;
  }
  
  export interface ItockaMerilnegaMesta {
    tocka: Itocka;
    vrednost: number;
    polmer: number;
    energentTip: string;
    stMerilnegaMesta: string;
    nickName: string;
  }

  // export interface IobjektObodInInfo {
  //   noviObodiObjekta: Itocka[][];
  //   idJavnegaObjekta: number;
  //   sifraJavnegaObjekta: string;
  //   naziv: string;
  //   center: google.maps.LatLng;
  // }

  

  
  export interface Ipoligon {
      idJavnegaObjekta: number;
      sifraJavnegaObjekta: string;
      naziv: string;
      center: google.maps.LatLng
  
    noviObodiObjekta: Itocka[][];
    noviObodiLatLng: google.maps.LatLngLiteral[][];
  }


